tools/msgdis.py

```diff
@@ -173,6 +173,1268 @@
+    0x6034 : "NA_SE_VO_LI_AUTO_JUMP_KID",
+    0x6035 : "NA_SE_VO_LI_MAGIC_NALE_KID",
+    0x6036 : "NA_SE_VO_LI_SURPRISE_KID",
+    0x6037 : "NA_SE_VO_LI_MAGIC_FROL_KID",
+    0x6038 : "NA_SE_VO_LI_PUSH_KID",
+    0x6039 : "NA_SE_VO_LI_HOOKSHOT_HANG_KID",
+    0x603A : "NA_SE_VO_LI_LAND_DAMAGE_S_KID",
+    0x603B : "NA_SE_VO_LI_NULL_0x1b_KID",
+    0x603C : "NA_SE_VO_LI_MAGIC_ATTACK_KID",
+    0x603D : "NA_SE_VO_BL_DOWN_KID",
+    0x603E : "NA_SE_VO_LI_DEMO_DAMAGE_KID",
+    0x603F : "NA_SE_VO_LI_ELECTRIC_SHOCK_LV_KID",
+    0x6040 : "NA_SE_VO_NAVY_ENEMY",
+    0x6041 : "NA_SE_VO_NAVY_HELLO",
+    0x6042 : "NA_SE_VO_NAVY_HEAR",
+    0x6043 : "NA_SE_VO_NAVY_CALL",
+    0x6044 : "NA_SE_VO_NA_HELLO_3",
+    0x6045 : "NA_SE_VO_DUMMY_0x45",
+    0x6046 : "NA_SE_VO_DUMMY_0x46",
+    0x6047 : "NA_SE_VO_DUMMY_0x47",
+    0x6048 : "NA_SE_VO_DUMMY_0x48",
+    0x6049 : "NA_SE_VO_DUMMY_0x49",
+    0x604A : "NA_SE_VO_DUMMY_0x4a",
+    0x604B : "NA_SE_VO_DUMMY_0x4b",
+    0x604C : "NA_SE_VO_DUMMY_0x4c",
+    0x604D : "NA_SE_VO_DUMMY_0x4d",
+    0x604E : "NA_SE_VO_DUMMY_0x4e",
+    0x604F : "NA_SE_VO_DUMMY_0x4f",
+    0x6050 : "NA_SE_VO_TA_SLEEP",
+    0x6051 : "NA_SE_VO_TA_SURPRISE",
+    0x6052 : "NA_SE_VO_TA_CRY_0",
+    0x6053 : "NA_SE_VO_TA_CRY_1",
+    0x6054 : "NA_SE_VO_IN_CRY_0",
+    0x6055 : "NA_SE_VO_IN_LOST",
+    0x6056 : "NA_SE_VO_IN_LASH_0",
+    0x6057 : "NA_SE_VO_IN_LASH_1",
+    0x6058 : "NA_SE_VO_FR_LAUGH_0",
+    0x6059 : "NA_SE_VO_FR_SMILE_0",
+    0x605A : "NA_SE_VO_NB_AGONY",
+    0x605B : "NA_SE_VO_NB_CRY_0",
+    0x605C : "NA_SE_VO_NB_NOTICE",
+    0x605D : "NA_SE_VO_NA_HELLO_0",
+    0x605E : "NA_SE_VO_NA_HELLO_1",
+    0x605F : "NA_SE_VO_NA_HELLO_2",
+    0x6060 : "NA_SE_VO_RT_CRASH",
+    0x6061 : "NA_SE_VO_RT_DISCOVER",
+    0x6062 : "NA_SE_VO_RT_FALL",
+    0x6063 : "NA_SE_VO_RT_LAUGH_0",
+    0x6064 : "NA_SE_VO_RT_LIFT",
+    0x6065 : "NA_SE_VO_RT_THROW",
+    0x6066 : "NA_SE_VO_RT_UNBALLANCE",
+    0x6067 : "NA_SE_VO_ST_DAMAGE",
+    0x6068 : "NA_SE_VO_ST_ATTACK",
+    0x6069 : "NA_SE_VO_Z0_HURRY",
+    0x606A : "NA_SE_VO_Z0_MEET",
+    0x606B : "NA_SE_VO_Z0_QUESTION",
+    0x606C : "NA_SE_VO_Z0_SIGH_0",
+    0x606D : "NA_SE_VO_Z0_SMILE_0",
+    0x606E : "NA_SE_VO_Z0_SURPRISE",
+    0x606F : "NA_SE_VO_Z0_THROW",
+    0x6070 : "NA_SE_VO_SK_CRY_0",
+    0x6071 : "NA_SE_VO_SK_CRY_1",
+    0x6072 : "NA_SE_VO_SK_CRASH",
+    0x6073 : "NA_SE_VO_SK_LAUGH",
+    0x6074 : "NA_SE_VO_SK_SHOUT",
+    0x6075 : "NA_SE_VO_Z1_CRY_0",
+    0x6076 : "NA_SE_VO_Z1_CRY_1",
+    0x6077 : "NA_SE_VO_Z1_OPENDOOR",
+    0x6078 : "NA_SE_VO_Z1_SURPRISE",
+    0x6079 : "NA_SE_VO_Z1_PAIN",
+    0x607A : "NA_SE_VO_KZ_MOVE",
+    0x607B : "NA_SE_VO_NB_LAUGH",
+    0x607C : "NA_SE_VO_IN_LAUGH",
+    0x607D : "NA_SE_VO_DUMMY_0x7d",
+    0x607E : "NA_SE_VO_DUMMY_0x7e",
+    0x607F : "NA_SE_VO_DUMMY_0x7f",
+}
+
 # From https://stackoverflow.com/questions/241327/remove-c-and-c-comments-using-python
 def remove_comments(text : str) -> str:
```

Comment:
Should be good to remove this function as well. It was only used for parsing the include tables I believe

Reply:
good catch, thanks